parm/gdas/atm_berror_bump_staging.yaml.j2

```diff
@@ -0,0 +1,23 @@
+{% set src_dir = BERROR_DATA_DIR %}
+{% set dest_dir = DATA ~ '/berror' %}
+{% set ftype_list = ['cor_rh', 'cor_rv', 'stddev'] %}
+{% set nprocs = ntiles * layout_x * layout_y %}
+
+mkdir:
+- '{{ dest_dir }}'
+
+copy:
+{% for ftype in ftype_list %}
+{% set prefix = BERROR_DATE | to_fv3time ~ '.' ~ ftype %}
+{% set fname = prefix ~ '.coupler.res' %}
+- ['{{ src_dir }}/{{ fname }}', '{{ dest_dir }}/{{ fname }}']
+{% for itile in range(1,7) %}
+{% set fname = prefix ~ '.fv_tracer.res.tile' ~  itile ~ '.nc' %}
+- ['{{ src_dir }}/{{ fname }}', '{{ dest_dir }}/{{ fname }}']
+{% endfor %}
+{% endfor %}
+
+{% for nn in range(1,nprocs+1) %}
+{% set fname = 'nicas_aero_nicas_local_%06d-%06d.nc' | format(nprocs, nn) %}
```

Comment:
this is the aero nicas but the filename is atm, probably need to rename the file as I don't think we have ATM BUMP files anywhere